app/uk/gov/hmrc/directdebitupdateemailfrontend/config/AppConfig.scala

```diff
@@ -35,6 +35,8 @@ class AppConfig @Inject() (config: Configuration) {
     val ddUpdateEmailFrontend: String = platformHost.getOrElse(config.get[String]("baseUrl.direct-debit-update-email-frontend-local"))
     val gg: String = config.get[String]("baseUrl.gg")
     val signOutUrl: String = config.get[String]("baseUrl.sign-out")
+    val contactFrontend: String = platformHost.getOrElse(config.get[String]("baseUrl.contact-frontend"))
+    def betaFeedbackUrl: String = s"${BaseUrl.contactFrontend}/contact/beta-feedback?service=direct-debit-frontend"
```

Comment:
this could be a `val` instead of a `def`

Reply:
whoops sorry yeah it is supposed to be, I was seeing weird behaviour with DI but turns out I just needed to clean compile after rebase